ajax/libs/simpleCookie/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "simplecookie",
+  "version": "1.0.1",
+  "description": "A small javascript utility to make handling cookies simple",
+  "filename": "simpleCookie.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tevko/simpleCookie.git"
+  },
+  "keywords": [
+    "javascript",
+    "cookie"
+  ],
+  "author": "Tim Evko",
+  "license": "ISC",
```

```suggestion
  "license": "MIT",
```
License in the package appears to be MIT